awx/asgi.py

```diff
@@ -9,7 +9,7 @@
 from awx import prepare_env, MODE
 prepare_env() # NOQA
 
-from channels.routing import get_default_application
+from channels.routing import get_default_application  # noqa
```

Comment:
@beeankha I wonder you could just move this up above `prepare_env()` to make flake8 happy (ln 8 probably).  

Unless there is some reason why we have prepare_env() being called before that import...

Reply:
It looks like it was this way when this got added in 2016 - https://github.com/ansible/awx/commit/b448621387f92ddb82eb0ed0cd0e9b37dc44d320